src/utils/log.ts

```diff
@@ -8,6 +8,7 @@ export type Logger = {
   warn: (message: any) => void;
   error: (message: any) => void;
   debug: (message: any) => void;
+  fatal: (message: any) => void;
```

Comment:
NITS: Please reorder levels: debug -> info -> warn -> error -> fatal.
Same for logger instances.